src/types/index.js

```diff
@@ -36,4 +36,16 @@ Object.keys(annotationTypes).forEach(key => {
   defaults.describe(`elements.${annotationTypes[key].id}`, {
     _fallback: 'plugins.annotation'
   });
+  if (key === 'line') {
```

Comment:
Placing these to `LineAnnotation.descriptors` should work.

Eg: (in line.js)

```js
LineAnnotation.descriptors = {
  arrowHeads: {
    _fallback: true
  }
}
```

Are the `.start` and `.end` descriptors actually needed?

Reply:
Removing them, it doesn't do any fallback, at least in my test case.

Reply:
but I moved in

```javascript
LineAnnotation.descriptors = {
  arrowHeads: {
    start: {
      _fallback: true
    },
    end: {
      _fallback: true
    },
    _fallback: true
  }
};
```

Reply:
done